test/runnable/extra-files/cdvecfillavx2.out

```diff
@@ -3,425 +3,358 @@
+   1:	c4 e2 79 79 07       	vpbroadcastw xmm0,WORD PTR [rdi]
+   6:	59                   	pop    rcx
+   7:	c3                   	ret    
 
 Disassembly of section .text._D9cdvecfill4testFkZNhG4k:
 
 0000000000000000 <_D9cdvecfill4testFkZNhG4k>:
-   0:	55                   	push   rbp
-   1:	48 8b ec             	mov    rbp,rsp
-   4:	c5 f9 6e c7          	vmovd  xmm0,edi
-   8:	c4 e2 79 58 c0       	vpbroadcastd xmm0,xmm0
-   d:	5d                   	pop    rbp
-   e:	c3                   	ret    
-	...
+   0:	50                   	push   rax
+   1:	c5 f9 6e c7          	vmovd  xmm0,edi
+   5:	c4 e2 79 58 c0       	vpbroadcastd xmm0,xmm0
+   a:	59                   	pop    rcx
+   b:	c3                   	ret    
 
 Disassembly of section .text._D9cdvecfill4testFPkZNhG4k:
 
 0000000000000000 <_D9cdvecfill4testFPkZNhG4k>:
-   0:	55                   	push   rbp
-   1:	48 8b ec             	mov    rbp,rsp
-   4:	c4 e2 79 58 07       	vpbroadcastd xmm0,DWORD PTR [rdi]
-   9:	5d                   	pop    rbp
-   a:	c3                   	ret    
-	...
+   0:	50                   	push   rax
+   1:	c4 e2 79 58 07       	vpbroadcastd xmm0,DWORD PTR [rdi]
+   6:	59                   	pop    rcx
+   7:	c3                   	ret    
 
 Disassembly of section .text._D9cdvecfill4testFiZNhG4i:
 
 0000000000000000 <_D9cdvecfill4testFiZNhG4i>:
-   0:	55                   	push   rbp
-   1:	48 8b ec             	mov    rbp,rsp
-   4:	c5 f9 6e c7          	vmovd  xmm0,edi
-   8:	c4 e2 79 58 c0       	vpbroadcastd xmm0,xmm0
-   d:	5d                   	pop    rbp
-   e:	c3                   	ret    
-	...
+   0:	50                   	push   rax
+   1:	c5 f9 6e c7          	vmovd  xmm0,edi
+   5:	c4 e2 79 58 c0       	vpbroadcastd xmm0,xmm0
+   a:	59                   	pop    rcx
+   b:	c3                   	ret    
 
 Disassembly of section .text._D9cdvecfill4testFPiZNhG4i:
 
 0000000000000000 <_D9cdvecfill4testFPiZNhG4i>:
-   0:	55                   	push   rbp
-   1:	48 8b ec             	mov    rbp,rsp
-   4:	c4 e2 79 58 07       	vpbroadcastd xmm0,DWORD PTR [rdi]
-   9:	5d                   	pop    rbp
-   a:	c3                   	ret    
-	...
+   0:	50                   	push   rax
+   1:	c4 e2 79 58 07       	vpbroadcastd xmm0,DWORD PTR [rdi]
+   6:	59                   	pop    rcx
+   7:	c3                   	ret    
 
 Disassembly of section .text._D9cdvecfill4testFmZNhG2m:
 
 0000000000000000 <_D9cdvecfill4testFmZNhG2m>:
-   0:	55                   	push   rbp
-   1:	48 8b ec             	mov    rbp,rsp
-   4:	c4 e1 f9 6e c7       	vmovq  xmm0,rdi
-   9:	c4 e2 79 59 c0       	vpbroadcastq xmm0,xmm0
-   e:	5d                   	pop    rbp
-   f:	c3                   	ret    
+   0:	50                   	push   rax
+   1:	c4 e1 f9 6e c7       	vmovq  xmm0,rdi
+   6:	c4 e2 79 59 c0       	vpbroadcastq xmm0,xmm0
+   b:	59                   	pop    rcx
+   c:	c3                   	ret    
+   d:	00 00                	add    BYTE PTR [rax],al
```

Comment:
Why are those two zero bytes written? (I assume that disassembly doesn't make sense as the `add` instructions occurs after `ret`.)

Reply:
alignment of code